tests/test_modis.py

```diff
@@ -662,6 +717,29 @@ def test_create(self):
 
         smtH5.filename.unlink()
 
+    # @patch.object(ModisSmoothH5, "read_chunked")
+    # @patch.object(ModisSmoothH5, "write_chunk", return_value=False)
+    # @patch("modape.modis.smooth.HDF5Base.read_chunked")
+    def test_smooth_export_import(self):  # mock_hdf5base_read, mocked_write, mocked_read):
```

Comment:
@valpesendorfer This test here...

Reply:
ok gotta admit this is a quite steep curve catching up again, and remembering wth I was thinking back then.

Can you say what exactly needs to be tested? Feel free to list the steps, that would be helpful

Reply:
Thanks for looking into this and refreshing your memory on the space-time chunking of data into the H5 files.

It would be great to have a test that documents-as-code:
1. Have mocked grids of NDVI values for n number of dates, for instance generated using `np.random.randint(300, 5000, size=( ??, ?? ))` and have this cube written into the H5
2. Run smoother on this cube, thus initializing the S-grid, written to H5
3. Export the S-grid to GeoTIFF
4. Then import again this GeoTIFF into an empty H5
5. Compare the S-grids to be exactly the same

If you could code step 1 + 2; I would be able to code 3 - 5

As you code step 1 + 2, it would be great if you could add comments (in the test, or perhaps in existing code) on how the chunking works

Reply:
@valpesendorfer 

Actually, if you could code only Step 1...

>     1. Have mocked grids of NDVI values for n number of dates, for instance generated using `np.random.randint(300, 5000, size=( ??, ?? ))` and have this cube written into the H5

... I should be able to code the rest of the steps for this test myself. I realize that this test is more _integration test_-like than unit test; but it would very well document the inner workings (data handling) of modape.